src/test/java/com/skhu/luxuryshop/product/controller/ProductControllerTest.java

```diff
@@ -0,0 +1,92 @@
+package com.skhu.luxuryshop.product.controller;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.skhu.luxuryshop.product.dto.ProductRequestDto;
+import com.skhu.luxuryshop.product.dto.ProductResponseDto;
+import com.skhu.luxuryshop.product.entity.ProductEntity;
+import com.skhu.luxuryshop.product.service.ProductService;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.ArgumentMatchers;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@WebMvcTest(ProductController.class)
+class ProductControllerTest {
+    @Autowired
+    private ObjectMapper objectMapper;
+    @Autowired
+    private MockMvc mockMvc;
+    @MockBean
+    private ProductService productService;
+
+    private ProductRequestDto product;
+    private ProductEntity productEntity;
+
+    @BeforeEach
+    void setUp() {
+        product = new ProductRequestDto("controller 테스트", "controller 테스트", 900, "controller 테스트", "controller 테스트");
+        productEntity = new ProductEntity(1L, "controller test", "controller", 900, "controller", "controller");
+    }
+
+    @Test
+    void 상품등록_Test() throws Exception {
+        ProductResponseDto productResponse = new ProductResponseDto(1L, product.getName(), product.getContent(), product.getPrice()
+                , product.getCategory(), product.getImageurl());
```

Comment:
![image](https://user-images.githubusercontent.com/33235211/115397594-3b73d700-a221-11eb-88f1-4123c1c59793.png)

줄바꿈 허용 위치도 컨벤션이 있슴니다! 별게다있죠??

```suggestion
        ProductResponseDto productResponse = new ProductResponseDto(1L, product.getName(), product.getContent(), product.getPrice(),
                product.getCategory(), product.getImageurl());
```